test/e2e/network/service.go

```diff
@@ -1339,8 +1339,15 @@ var _ = SIGDescribe("Services", func() {
 
 	It("should be able to create an internal type load balancer [Slow]", func() {
 		framework.SkipUnlessProviderIs("azure", "gke", "gce")
+		if framework.ProviderIs("gke", "gce") {
+			framework.SkipUnlessNodeCountIsAtMost(framework.GCPMaxInstancesInInstanceGroup)
+		}
 
 		createTimeout := framework.LoadBalancerCreateTimeoutDefault
+		if nodes := framework.GetReadySchedulableNodesOrDie(cs); len(nodes.Items) > framework.LargeClusterMinNodesNumber {
+			createTimeout = framework.LoadBalancerCreateTimeoutLarge
```

Comment:
Does this mean we were not actually using the 2hr timeout till now?

Reply:
This is specifically for ILB test. For the other LB tests we are using 2hr :)